src/bin/install.rs

```diff
@@ -37,7 +37,9 @@ pub struct Options {
 }
 
 pub const USAGE: &'static str = "
-Install a Rust binary
+Install a Rust binary, for example, additional Cargo commands
+
+(If you're an npm user, please note this is completely unrelated to `npm install`. Please see `cargo build --help`.)
```

Comment:
I would think that ‘npm install -g’ is at least partially similar to ‘cargo install’? At least, I use -g to install node utilities for the current user, but I may be missusing this flag :-)

Reply:
this is a good point, `cargo install` *is* lik` `npm install -g`